Frontend/src/Components/PPC_MPPT/PPC_MPPT.js

```diff
@@ -34,43 +37,134 @@ export default function PPC_MPPT(props) {
-                    unit="&#8451;"
-                    min={CONSTANTS.string1_temp.MIN}
-                    max={CONSTANTS.string1_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string2_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string2_temp.MIN}
-                    max={CONSTANTS.string2_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string3_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string3_temp.MIN}
-                    max={CONSTANTS.string3_temp.MAX}
-                />
+                <Grid
+                    templateColumns="repeat(4, 1fr)"
+                    templateRows="repeat(4, 1fr)"
+                    gap={1}
+                    overflowY='scroll'
+                >
+                    <GridItem colSpan={1}></GridItem>
+                    <div>Temp(ºC)</div>
+                    <div>Voltage in (V)</div>
+                    <div>Current in (A)</div>
+                    <GridItem fontSize={12}>String 1</GridItem>
+                    <GridItem colSpan={0.1}>
+                        <RangeCell
+                            fontSize='1vw'
+                            //label='Cell String 1 Temp'
+                            data={props.data?.string1_temp[0] ?? -1.0}
+                            digits={3}
+                            //unit="&#8451;"
+                            min={CONSTANTS.string1_temp.MIN}
+                            max={CONSTANTS.string1_temp.MAX}
+                        />
+                    </GridItem>
+                    <GridItem>
+                        <RangeCell
+                            fontSize='1vw'
+                            //label='Cell String 1 Temp'
+                            data={props.data?.string1_V_in[0] ?? -1.0}
+                            digits={3}
+                            //unit="&#8451;"
+                            min={CONSTANTS.string1_V_in.MIN}
+                            max={CONSTANTS.string1_V_in.MAX}
+                        />
+                    </GridItem>
+                    <GridItem>
+                        <RangeCell
+                            fontSize='1vw'
+                            //label='Cell String 1 Temp'
+                            data={props.data?.string1_I_in[0] ?? -1.0}
+                            digits={3}
+                            //unit="&#8451;"
+                            min={CONSTANTS.string1_I_in.MIN}
+                            max={CONSTANTS.string1_I_in.MAX}
+                        />
+                    </GridItem>
+
+                    <GridItem fontSize={12}>String 2</GridItem>
```

Comment:
Would it make any sense to put this text in a <Text> with sm/md/lg sizes so that any changes to the global CSS affect this too?